Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/RequirementNodeData.xml

```diff
@@ -6980,4 +6980,49 @@
         <Flags index="TechTreeCheat" value="0"/>
         <Count Link="AP_SILiberatorDefenderMode" State="CompleteOnly"/>
     </CRequirementCountUpgrade>
+    <CRequirementAnd id="AP_AndGTECountUpgradeAP_AberrationBanelingLaunchCompleteOnly1LTCountUnitAP_BanelingLaunchAmmoQueuedOrBetterAtUnit1">
+        <OperandArray value="AP_GTECountUpgradeAP_AberrationBanelingLaunchCompleteOnly1"/>
+        <OperandArray value="AP_LTCountUnitAP_BanelingLaunchAmmoQueuedOrBetterAtUnit1"/>
+    </CRequirementAnd>
+    <CRequirementAnd id="AP_AndGTECountUpgradeAP_AberrationBanelingLaunchLevel2CompleteOnly1GTECountUnitAP_BanelingLaunchAmmoCompleteOnlyAtUnit3">
+        <Tooltip value="RequirementNode/Tooltip/AP_AP_AndGTECountUpgradeAP_AberrationBanelingLaunchLevel2CompleteOnly1GTECountUnitAP_BanelingLaunchAmmoCompleteOnlyAtUnit3"/>

```

Comment:
This tooltip doesn't point to a key in GameStrings.txt, and the requirementNode doesn't look like it's used anywhere that would have it show up in the UI. This line should probably be removed.

Reply:
fixed